src/modules/disk.cpp

```diff
@@ -73,6 +73,7 @@ auto waybar::modules::Disk::update() -> void {
       ));
   }
   event_box_.show();
+  getState((stats.f_blocks - stats.f_bavail) * 100 / stats.f_blocks);
```

Comment:
Could save the operation into a variable in order to avoid computing it twice 😄

Reply:
Done 😃